proto/api/api.proto

```diff
@@ -223,11 +226,27 @@ message SuggestGasPriceResponse {
   uint64 gasPrice = 1;
 }
 
+// To be deprecated
 message EstimateGasForActionRequest {
+  iotextypes.Action action = 1;
```

Comment:
won't this broke the existing api?

Reply:
It won't,this is the original api that is using right now,but it's for sealed action,we add new one for not sealed execution and transfer